tests/integration/test_lock_resolver.py

```diff
@@ -523,3 +524,40 @@ def assert_dists(
     assert_requirements(pex_info)
     assert_dists(pex_info, "root", "middle_man_with_extras", "A", "extra1", "B", "extra2", "C")
     subprocess.check_call(args=test_pex_args)
+
+
+def test_resolve_wheel_files(tmpdir):
+    # type: (Any) -> None
+
+    lock = os.path.join(str(tmpdir), "lock")
+    # N.B.: We choose ansicolors 1.1.8 since it works with all Pythons and has a universal wheel
+    # published on PyPI and cowsay 5.0 since it also works with all Pythons and only has an sdist
+    # published on PyPI. This combination ensures the resolve process can handle both building
+    # wheels (cowsay stresses this) and using pre-existing ones (ansicolors stresses this).
+    run_pex3("lock", "create", "ansicolors==1.1.8", "cowsay==5.0", "-o", lock).assert_success()
+
+    pex = os.path.join(str(tmpdir), "pex")
+    exe = os.path.join(str(tmpdir), "exe")
+    with open(exe, "w") as fp:
+        fp.write("import colors; print(colors.blue('Moo?'))")
```

Comment:
I assume the `import colours` is meaning to validate that `ansicolors` is actually installed in a useful way. Should there be an  `import cowsay` too?

Reply:
Fixed.